sdk/Lusid.Sdk.Tests/PropertyBasedConverterTests.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Utilities;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests
+{
+    [TestFixture]
+    public class PropertyBasedConverterTests
+    {
+
+        [DataContract]
+        class SimpleObject
```

Comment:
Would it make sense to have these inherit from IDictionary to simulate the actual use case?

Reply:
done